content/en/continuous deployment.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Please consider to rename this file to `continuous_deployment.md`, for the consistency. 😊

([Files in `content/en/`](https://github.com/cncf/glossary/tree/main/content/en))

Reply:
I have made the changes @jihoon-seo!